docs/articles_en/about-openvino/system-requirements.rst

```diff
@@ -152,7 +151,7 @@ Operating systems and developer environment
 
    .. tab-item:: macOS
 
-      * macOS 10.15 and above
+      * macOS 12.6 and above
```

Comment:
why do we need it?
we built binaries for 10.12 x86_64 and 11.0 for arm64
See
![image](https://github.com/openvinotoolkit/openvino/assets/2566854/62c694a6-4456-45fc-9b0d-32cdeaaad27c)

I propose to use versions in wheels.

Reply:
We need this because 11.x and early versions are not supported by Apple.

Reply:
but supported by OpenVINO 😄 
why do we need to replicate what Apple supports?

Reply:
No, it's not supported by OpenVINO as it's no longer validated on OpenVINO and we will not fix any issues related to compatibility on those. 
It may still work on OpenVINO for some time, but it's definitely should not be used in prod environment.
And yes, we need to rely on support of OS by vendor as we don't want OpenVINO to run in insecure environment.